wpimath/src/main/native/include/frc/MathUtil.h

```diff
@@ -55,4 +55,16 @@ constexpr units::radian_t AngleModulus(units::radian_t angle) {
                                        units::radian_t{wpi::numbers::pi});
 }
 
+/**
+ * Adds a scalar value to whatever is input
+ *
+ * @param value The number the scalar is added to.
+ * @param scalar The magnitude that is added to the value
+ * @return The value with the scalar added.
+ */
+WPILIB_DLLEXPORT
+double AddScalar(double value, double scalar) {
+  return std::copysign(std::abs(value) + std::abs(scalar), value);
+}
```

Comment:
You can only put the definition in the header if it's a template function or constexpr function.